coralproject/talk-fe
# Embed code

## What does this PR do?

Adds a page to coral-admin with an embed code and hosts the scripts which make that code function.

## How do I test this PR?
1. From the talk-fe directory: `cd packages/coral-embed-stream
2.`npm run build` (Note, we should discuss how to automate this process as part of a broader build script, but this seems like a larger conversation.)
3. `cd ../coral-admin`
4. `npm start`
5. In a web browser, navigate to http://localhost:3142/embedlink/
6. Copy the link into a simple webpage.
7. Webpage should include an embedded comment stream.
